src/main/java/com/autotune/database/helper/DBHelpers.java

```diff
@@ -300,6 +300,9 @@ public static KruizeExperimentEntry convertCreateAPIObjToExperimentDBObj(CreateE
                     kruizeExperimentEntry.setStatus(AnalyzerConstants.ExperimentStatus.IN_PROGRESS);
                     kruizeExperimentEntry.setMeta_data(null);
                     kruizeExperimentEntry.setDatasource(null);
+                    if (apiObject.getTargetCluster().equalsIgnoreCase(AnalyzerConstants.LOCAL)) {
```

Comment:
Could you reconsider the approach? The distinction between container and namespace experiments should be handled in a unified manner, with the workflow branching accordingly. Having too many if-else conditions reduces readability and makes debugging much harder